prow/report/report.go

```diff
@@ -44,10 +44,6 @@ type GithubClient interface {
 	EditComment(org, repo string, ID int, comment string) error
 }
 
-type configAgent interface {
-	Config() *config.Config
```

Comment:
We will likely need this for other reasons inside Report but I am removing it for now.